web/pages/embed/[username]/[contractSlug].tsx

```diff
@@ -115,7 +115,10 @@ function ContractEmbed(props: { contract: Contract; bets: Bet[] }) {
 
           {isBinary && (
             <Row className="items-center gap-4">
-              <BetRow contract={contract as BinaryContract} />
+              <BetRow
+                contract={contract as BinaryContract}
+                betPanelClassName="scale-75"
```

Comment:
Are you sure we need to scale this down? It makes the font small and is inconsistent. Plus, if you have shares already, you can still access the sell panel, which is not scaled.

I would advise finding another solution if it needs to be smaller. But the panel is designed to work on smaller mobile devices, so I would think it should be good?